internal/server/templates/index.html

```diff
@@ -1,116 +1,262 @@
+      display: flex;
+      justify-content: space-between;
+    }
+
+    .token-entry {
+      flex-grow: 1;
+      margin-right: 0.6rem;
+    }
+
+    .token-textarea {
+      box-sizing: border-box;
+      resize: vertical;
+      width: 100%;
+      height: 100%;
+    }
+
+    .token-info {
+      white-space: nowrap;
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+    }
+
+    .token-info button {
+      width: 100%;
+      padding: 0.1rem 1.0rem;
+      cursor: pointer;
+    }
+
+    .token-info button:first-child {
+      margin-top: 0.1rem;
+      margin-bottom: 0.4rem;
+    }
+
+    .response-container {
+      width: 100%;
+      flex: 1;
+      overflow: auto;
+      border: 0.1rem inset #ccc;
+      padding-block: 0.2rem;
+      padding-inline: 0.4rem;
+      user-select: text;
+    }
+  </style>
 </head>
+
 <body>
-<div class="page-container">
-<div class="controls-container">
-<form x-data="formHandler" @submit.prevent="handleSubmit">
-<label for="url">Enter a URL:</label>
-<select title="URL Type" x-model="formAction">
-	<option value="/extract">Page</option>
-	<option value="/extract/headless">Headless</option>
-	<option value="/feed">Feed</option>
-</select>
-<input type="url" name="url" id="url" value="https://" size="96" maxlength="200" pattern="https?://.*" required title="URL">
-<input type="submit" value="Hit It">
-<!-- <label for="token">Token:</label> -->
-<input type="hidden" name="token" size="101" maxlength="255" value="{{AuthToken}}" placeholder="Enter your access token">
-<input type="hidden" name="pp" value="1">
-</form>
-</div><!-- controls-container -->
-<div class="response-container">
-	<pre x-ref="responseContent" id="responseContent"></pre>
-</div> 
-</div><!-- page-container -->
-<script type="text/javascript">
-	function formHandler() {
-		return {
-			formAction: '/extract',
-			async handleSubmit(event) {
-				const form = event.target;
-				const headers = new Headers();
-				const token = form.token.value;
-				if (token) {
-					headers.append('Authorization', `Bearer ${token}`);
-				}
-				const formData = new FormData(form);
-				formData.delete('token');
-				try { 
-					const response = await fetch(this.formAction, {
-						method: 'POST',
-						headers: headers,
-						body: formData
-					})
-					
-					if (response.ok) {
-						const json = await response.json();
-						const jsonStr = JSON.stringify(json, null, 2);
-						this.updateContent(jsonStr);
-					} else {
-						const text = await response.text();
-						this.updateContent(`Error ${response.status}:\n${text}`);
-						throw new Error(`${response.status} - ${text}`);
-					}
-				} catch (error) {
-					console.error(error);
-				}
-			},
-			updateContent(content) {
-				document.getElementById('responseContent').textContent = content;
-				// following should work but doesn't
-				// this.$refs.responseContent.textContent = content;
-			}
-		}
-	}
-</script>
+  <div class="page-container">
```

Comment:
There _are_ a bunch of changes in this block, but the reason it's showing up as a a complete block diff is because I fixed the formatting.